### ISSUE TITLE
Trips Informations

### ISSUE THREAD

**Issue by jvanderweken** [Dielee/volvo2mqtt#149]
Hi

is it possible to import the Trips? 

Like in the Volvo App or like in the EVEE App?

Both takes also just informations of the API?

**Comment by Dielee** [comment]
No, sorry, there is no endpoint available containing this data.
